### PR TITLE
V0.1.2

### PR DESCRIPTION
**Pull request by ray-rock**
### What's new?
1. Fixed default storage config (#8)
2. ~~Changes default vpn routing, adds documentation for whether one should include their indexer or only tunnel qbittorrent. (#9)~~
3. ~~Fix an issue where other containers might start before gluetun, leaving them networkless and unable to be used. (#10)~~
4. moved vpn configuration to ```config/gluetun/vpn-config.env```
5. Made changes to reverse proxy to make it functional (#19)
    - Added documentation on creating DNS entries for reverse proxy
6. Massive improvements to documentation/readme

<sup>*Stricken entries unfurled into larger issues.</sup>

### To-Do
- Change exposed ports
- Add SSL termination

Issues #9 and #10 require more analysis.

### CODE REVIEW

**Comment by ray-rock** [comment]
Must fully test before merge.

**Review by ray-rock — COMMENTED** [review]
readme.md does not credit authors of deemix, gluetun, or -arr apps.

**Review by ray-rock — COMMENTED** [review]
alt/jackett-no-prowlarr/docker-compose.yml is out of sync with main. Jackett can be removed from docker-compose.yml before merge.

File: docker-compose.yml [hunk 1]
```diff
@@ -58,8 +85,39 @@ services:
             - WEBUI_PORT=8080
         volumes:
             - ./config/qbt:/config
-            - downloads:/downloads
+            - ./data:/data
```
Comment (stevietv) [inline docker-compose.yml:88]: best practice here for qbittorrent would be to mount
`- ./data/torrents:/data/torrents`
qbit doesn't need access to everything, only where it downloads to temporarily.

**Review by stevietv — COMMENTED** [review]
see my review regarding mount path for qbit

**Review by ray-rock — COMMENTED** [review]
Latest commits satisfy issues brought up in review. This pr may be merged to main.

File: docker-compose.yml (hunk 1 above)
Comment (ray-rock) [reply docker-compose.yml:88]: Totally missed this, I'll revise for v0.2.3

Comment (ray-rock) [reply docker-compose.yml:88]: #27

**Pull request merged**